packages/modal/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "devDependencies": {
+    "bower": "1.8.2",
+    "grunt": "1.0.1",
+    "grunt-contrib-clean": "1.1.0",
+    "grunt-contrib-coffee": "1.0.0",
+    "grunt-contrib-connect": "1.0.2",
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-contrib-jasmine": "1.1.0",
+    "grunt-contrib-less": "1.4.1",
+    "grunt-contrib-watch": "1.0.0",
+    "grunt-open": "0.2.3",
+    "load-grunt-tasks": "3.5.2"
+  },
+  "files": ["app/script/", "app/style/", "dist/script/", "dist/style/"],
+  "license": "GPL-3.0",
+  "name": "@wireapp/modal",
```

Comment:
After talking to @AndyTheGiant & @Yserz we should brand that module with the name `@wireapp/jquery-modal`.

Reply:
agree